lazycatlabs/flutter_auth_app
# feat: use flutter native splash

## What it does

- use [Flutter Native Splash](https://pub.dev/packages/flutter_native_splash) library

## How to test

1. clone the project
2. run the project with different appearance modes (dark/light)
3. observe the view

## Screenshot

https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/f7f0e78f-e982-4143-8e22-1e488b55cca2
